src/rules/memberOrderingRule.ts

```diff
@@ -78,12 +78,16 @@ export class MemberOrderingWalker extends Lint.RuleWalker {
     }
 
     public visitPropertyDeclaration(node: ts.PropertyDeclaration) {
-        this.checkModifiersAndSetPrevious(node, getModifiers(false, node.modifiers));
+        const { initializer } = node;
+        const isFunction = initializer != null
+            && (initializer.kind === ts.SyntaxKind.ArrowFunction || initializer.kind === ts.SyntaxKind.FunctionExpression);
+        this.checkModifiersAndSetPrevious(node, getModifiers(isFunction, node.modifiers));
         super.visitPropertyDeclaration(node);
     }
 
-    public visitPropertySignature(node: ts.Node) {
-        this.checkModifiersAndSetPrevious(node, getModifiers(false, node.modifiers));
+    public visitPropertySignature(node: ts.PropertyDeclaration) {
```

Comment:
Looks like it is discussed here: https://github.com/Microsoft/TypeScript/issues/3833

`PropertySignature` nodes will be visited when linting interfaces. But looking at that code snippet from the parser, `ts.PropertyDeclaration` is probably safe enough to use as the typedef here. I guess we'll rely on testing :)